meta-mender-core/recipes-mender/mender-client/mender_4.0.5.bb

```diff
@@ -0,0 +1,37 @@
+require mender-client-cpp.inc
```

Comment:
The other recipes also require `mender_4.x.inc`, is that not needed here?

Reply:
Good catch! Yes, that is indeed needed here. I made these recipes before #2213 and later I didn't update.

Fixed now, thanks!